security/clamav/src/opnsense/service/templates/OPNsense/ClamAV/clamav_clamd

```diff
@@ -0,0 +1,6 @@
+{% if helpers.exists('OPNsense.clamav.general.enabled') and OPNsense.clamav.general.enabled == '1' %}
+clamav_opnsense_bootup_run="/usr/local/opnsense/scripts/OPNsense/ClamAV/setup.sh"
```

Comment:
I think this only works if named clamav_clamd_opnsense_bootup_run, "_opnsense_bootup_run" is the suffix